rust/processor/src/db/postgres/migrations/2025-05-01-025837_optimize_price_feed/up.sql

```diff
@@ -54,7 +54,7 @@ with_prices AS (
         END AS open_price_q64,
         latest_swap.last_swap_avg_execution_price_q64 AS close_price_q64
     FROM markets
-    INNER JOIN market_latest_state_event AS latest_swap ON markets.market_id = latest_swap.market_id
+    INNER JOIN market_state AS latest_swap ON markets.market_id = latest_swap.market_id
```

Comment:
Just noting here that we can indeed update in-place a migration that is on the alpha deployment because of the `DROP VIEW IF EXISTS price_feed;` statement at the top

- [ ] @xbtmatt Acknowledge this by resolving this comment